java-client/src/main/java/org/opensearch/client/opensearch/OpenSearchAsyncClient.java

```diff
@@ -116,6 +116,12 @@
 import org.opensearch.client.opensearch.core.UpdateByQueryRethrottleResponse;
 import org.opensearch.client.opensearch.core.UpdateRequest;
 import org.opensearch.client.opensearch.core.UpdateResponse;
+import org.opensearch.client.opensearch.core.point_in_time.CreatePointInTimeRequest;
```

Comment:
I think the convention other clients follow [1] is to use `pit` instead of longer `point_in_time` version 

[1] https://github.com/opensearch-project/opensearch-rs/pull/157/files

Reply:
I think you're right. I updated the namespace to be `pit` instead of `point_in_time`